crates/orderbook/openapi.yml

```diff
@@ -593,6 +598,31 @@ components:
       required:
         - expiration
         - amount
+    OnchainOrderData:
+      description: |
+        Provides the additional data for onchain placed orders
+      type: object
+      properties:
+        onchainUser:
+          description: |
+            If orders are placed as onchain orders, the owner of the order might 
+            be a smart contract, but not the user placing the order. The 
+            actual user will be provided in this field
+          $ref: "#/components/schemas/Address"
+        placementError:
+          description: |
+           Describes the error, if the order placement was not 
+           successful. This could happen, for example, if the 
+           valid_to is too high, or no valid quote was found or generated
+          type: string
+          enum:
+            [
+              InvalidQuoteId,
+              InvalidValidTo,
```

Comment:
Aren't this errors ETH Flow logic (instead of generic smart orders logic?)

Reply:
It's mixed. The quote is part of generic smart orders and validTo would be handled by specific components.

Reply:
What will be the meaning exactly of `InvalidValidTo`? What are the checks we are doing? (should it be more self explanatory, like: `ValidToExpired`